icechunk-python/tests/test_stateful_repo_ops.py

```diff
@@ -101,6 +102,25 @@ class TagModel:
     # message: str | None
 
 
+@dataclass
+class BranchModel:
```

Comment:
nice! though isn't this basically `list` with an extra `head` property?

Reply:
yes. at first i had more. Then simplified but still thought that in the future we would add more. should i go back to just list? Being able to say `.head` is pretty nice